Python/Image_Object_Detector/readme.md

```diff
@@ -0,0 +1,39 @@
+## Image Object Detector Script
+
+A python script that highlights a template that exists inside a provided image by utilizing computer vision.
+
+## Tech Stack
+
+* Python
+	* numpy
+	* imutils
+	* opencv
+
+## Run Local
+
+* Clone this repository
+
+```bash
+git clone https://github.com/abhijeet007rocks8/Dev-Scripts
+```
+
+* Change Directory
+
+```bash
+cd Dev-Scripts/"Python/Image_Object_Detector"
```

Comment:
why everone is adding " " inverted commas while changing directory .

Reply:
Some Directories has a white-space in between which results in erros while changing directory in the CLI. The CLI handles this automatically by putting these inverted commas to facilates its processing.

Reply:
I can remove it on this case as there is no white-spaces tho. but as of why? it's an auto complete from the windows CMD to be specific. Linux uses `\ ` instead for every white-space.

Reply:
ok 👍